impeller/renderer/backend/metal/compute_pass_mtl.mm

```diff
@@ -55,6 +55,8 @@
     return false;
   }
 
+  FML_DCHECK(!grid_size_.IsEmpty() && !thread_group_size_.IsEmpty());
```

Comment:
We use VALIDATION_LOG for such things and return false from here maybe.

Reply:
That's too strong. VALIDATION_LOG will kill the process. An application could get here by mistake and should fizzle instead of dying.

Reply:
(but a test should die)